src/styles/blocks/start-day-month.scss

```diff
@@ -0,0 +1,7 @@
+.month__day{
+   @each $name,$scope in $day-week { 
+     &--start-day--#{$name}:first-child {
+       margin-left: $scope;
+    }
+  } 
+}
```

Comment:
move this to rules or functions scss file, no need to create a separate file for each at-rule